Bottom: 609d30d85efb1015d5042ce3700df064e8fd17f6
Top:    10a42f1b40ac5992f5904eddc724a33e7948f267
Author: Robert Berger <robert.berger@ReliableEmbeddedSystems.com>
Date:   2017-10-03 23:58:47 +0300

rocko: linux-yocto-custom.bb no more double inclusions

Signed-off-by: Robert Berger <robert.berger@ReliableEmbeddedSystems.com>


---

diff --git a/scripts/lib/bsp/substrate/target/arch/common/recipes-kernel/linux/linux-yocto-custom.bb b/scripts/lib/bsp/substrate/target/arch/common/recipes-kernel/linux/linux-yocto-custom.bb
index 3e367c6..e3ab09c 100644
--- a/scripts/lib/bsp/substrate/target/arch/common/recipes-kernel/linux/linux-yocto-custom.bb
+++ b/scripts/lib/bsp/substrate/target/arch/common/recipes-kernel/linux/linux-yocto-custom.bb
@@ -44,13 +44,10 @@ SRC_URI += "file://defconfig"
 
 SRC_URI += "file://{{=machine}}.scc \
             file://{{=machine}}.cfg \
+            file://{{=machine}}-user-config.cfg \
+            file://{{=machine}}-user-patches.scc \
             file://{{=machine}}-user-features.scc \
            "
-# {{=machine}}-user-config.cfg and
-# {{=machine}}-user-patches.scc are
-# included by {{=machine}}.scc
-# removed from SRC_URI to avoid double inclusion
-# and like this avoiding applying same patch twice
 
 {{ if kernel_choice == "custom" and custom_kernel_need_kbranch == "y" and custom_kernel_kbranch and custom_kernel_kbranch != "master": }}
 KBRANCH = "{{=custom_kernel_kbranch}}"
